server/portal/settings/settings_default.py

```diff
@@ -49,22 +47,16 @@
 # DJANGO APP: WORKSPACE
 ########################
 
-_PORTAL_APPS_METADATA_NAMES = ["frontera_apps", "frontera_apps_dev"]
+_PORTAL_APPS_METADATA_NAMES = []
 _PORTAL_ALLOCATION = 'TACC-ACI'
-_PORTAL_APPS_DEFAULT_TAB = 'Data Processing'
+_PORTAL_APPS_DEFAULT_TAB = ''
```

Comment:
can this be dropped?

Reply:
Ah I think the use of this setting was lost at some point before this PR. I'll add a task to bring use of it back in action, this time in `_WORKBENCH_SETTINGS`